### PR TITLE
Add v1beta3 removal notes to UPGRADE.md

### LINKED ISSUE #5232: UPGRADE.md needs update to indicate removed storage levels
We no longer support `v1beta3` as a storage level.  This has impacts on `master-config.yaml` files and the upgrade path should be described for rolling updates. 

@ironcladlou

### PR DESCRIPTION
**Pull request by ironcladlou**
Closes #5232

### CODE REVIEW

**Comment by ironcladlou** [comment]
cc @deads2k @liggitt @smarterclayton

File: UPGRADE.md [hunk 1]
```diff
@@ -34,3 +34,15 @@ At that time, the openshift docker registry image must be upgraded in order to c
...
+Existing `v1beta3`   resources stored in etcd will still be readable and
+automatically converted    to `v1` by the master on first mutation. Existing
+`v1beta3` resources stored on disk are still readable by the    `oc` client
+and will be automatically converted to `v1` for transmission to the master.
+
+  OpenShift master configuration files will need updated to remove `v1beta3`
+references:
+
+  1. The `etcdStorageConfig.openShiftStorageVersion` field value should be `v1`.
+  1. The `apiLevels` field should contain only `v1`.
```
Comment (deads2k) [inline UPGRADE.md]: Also the kube `apilevels`

**Comment by deads2k** [comment]
Minor update.  lgtm otherwise.

@sdodson This should probably make it into release notes.
@jwforres Did the webconsole ever support an HA mode of any kind?  This could produce some weird results.  With multiple API servers, sometimes you'd get a valid response and sometimes you wouldn't.

@smarterclayton different `v1` apis are different.  In a rolling update, old masters that read and then write may remove fields that new masters have allowed.  Do we have any way to keep that from happening?

File: UPGRADE.md (hunk 1 above)
Comment (ironcladlou) [inline UPGRADE.md]: Added notes for the kube field, thanks!

**Comment by liggitt** [comment]
Note about oc clients <= 1.0.4 needing to pass `--api-version=v1`?

**Comment by ironcladlou** [comment]
>Note about oc clients <= 1.0.4 needing to pass --api-version=v1?
Done.

**Comment by deads2k** [comment]
[merge]

File: UPGRADE.md [hunk 2]
```diff
@@ -34,3 +34,19 @@ At that time, the openshift docker registry image must be upgraded in order to c
...
+1. The `v1beta3` API version is being removed in Origin 1.1 (OSE 3.1).
+Existing `v1beta3` resources stored in etcd will still be readable and
+automatically converted to `v1` by the master on first mutation. Existing
+`v1beta3` resources stored on disk are still readable by the `oc` client
+and will be automatically converted to `v1` for transmission to the master.
+
+  OpenShift master configuration files will need updated to remove `v1beta3`
+references:
+
+  1. The `etcdStorageConfig.openShiftStorageVersion` field value should be `v1`.
```
Comment (liggitt) [inline UPGRADE.md]: need `etcdStorageConfig.kubernetesStorageVersion` set to v1 also

Comment (ironcladlou) [inline UPGRADE.md]: Added

**Pull request merged**